classes/book.rb

```diff
@@ -7,4 +7,8 @@ def initialize(title, author)
     @author = author
     @rentals = []
   end
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make all your commit messages capitalized to make your pull request descriptive and professional 👍